lib/main.dart

```diff
@@ -40,9 +40,6 @@ class MainScreen extends StatelessWidget {
         ElevatedButton.styleFrom(textStyle: const TextStyle(fontSize: 20));
 
     return Scaffold(
-      appBar: AppBar(
-        title: const Text('Home Page'),
-      ),
```

Comment:
Was this change intentional?

Reply:
No, I forgot to undo this. Was testing it out.